Cargo.toml

```
@@ -129,6 +129,8 @@ prefix-trie = { version = "0.8.1", default-features = false, features = [] }
 pretty_assertions = { version = "1.4.1", default-features = false, features = ["std"] }
 priority-queue = { version = "2.7.0", default-features = false, features = [] }
 procfs = { version = "0.18.0", default-features = false, features = [] }
+pyroscope = { version = "0.5.4", default-features = false, features = [] }
+pyroscope_pprofrs = { version = "0.2.10", default-features = false, features = [] }
```

Version mismatch: The workspace dependency specifies `version = "0.5.4"` but Cargo.lock shows `version = "0.5.8"` was actually resolved. This suggests the version constraint is working as intended (allowing patch updates), but it's worth noting.

Also, there's a minor spacing inconsistency - this line is missing a space after the opening brace. For consistency with the rest of the file:

```suggestion
pyroscope_pprofrs = { version = "0.2.10", default-features = false, features = [] }
```